src/wallet/spend.cpp

```diff
@@ -1098,7 +1099,7 @@ static util::Result<CreatedTransactionResult> CreateTransactionInternal(
         if (IsDust(txout, wallet.chain().relayDustFee())) {
             return util::Error{_("Transaction amount too small")};
         }
-        txNew.vout.push_back(txout);
+        txNew.vout.emplace_back(txout);
```

Comment:
This will just be forwarded to the copy ctor. `push_back(std::move(txout))` would be more correct (and clear).

Reply:
I've added benchmarks to verify:
```
|               ns/op |                op/s |    err% |     total | benchmark
|--------------------:|--------------------:|--------:|----------:|:----------
|            2,089.91 |          478,489.15 |    0.1% |      1.10 | `BenchmarkAddOutputsEmplaceBackReserve`
|            1,257.81 |          795,031.71 |    0.1% |      1.10 | `BenchmarkAddOutputsInPlaceEmplaceBackReserve`
|            2,594.12 |          385,487.12 |    0.2% |      1.10 | `BenchmarkAddOutputsPushBack`
|            1,667.85 |          599,573.52 |    0.1% |      1.10 | `BenchmarkAddOutputsPushBackMoveReserve`
|            2,089.21 |          478,649.26 |    0.1% |      1.10 | `BenchmarkAddOutputsPushBackReserve`
```

ended up adding it directly to `txNew.vout` which is both faster and simpler - what do you think?